samples/bot-adaptive-card-actions/csharp/README.md

```diff
@@ -0,0 +1,128 @@
+---
+page_type: sample
+description: Demonstrating the feature where user can send adaptivecard actions using bot.
+products:
+- office-teams
+- office
+- office-365
+languages:
+- csharp
+extensions:
+ contentType: samples
+ createdDate: "12/27/2022 12:30:00 PM"
+urlFragment: officedev-microsoft-teams-samples-bot-adaptivecard-actions-csharp
+---
+
+# Get Adaptive Card and Suggested Actions
+
+This sample shows the feature where user can send Adaptive Card actions using bot.
+
+## Interaction with app
+
+![Module](AdaptiveCardActions/Images/AdaptiveCardActions.gif)
+
+## Prerequisites
+
+- Microsoft Teams is installed and you have an account
+- [.NET SDK](https://dotnet.microsoft.com/download) version 6.0
+- [ngrok](https://ngrok.com/download) or equivalent tunnelling solution
+
+- [Teams](https://teams.microsoft.com) Microsoft Teams is installed and you have an account
+
+## Setup
+    
+1. Register a new application in the [Azure Active Directory – App Registrations](https://go.microsoft.com/fwlink/?linkid=2083908) portal.
+
+2. Setup for Bot
+	- Register a AAD aap registration in Azure portal.
+	- Also, register a bot with Azure Bot Service, following the instructions [here](https://docs.microsoft.com/en-us/azure/bot-service/bot-service-quickstart-registration?view=azure-bot-service-3.0).
+	- Ensure that you've [enabled the Teams Channel](https://docs.microsoft.com/en-us/azure/bot-service/channel-connect-teams?view=azure-bot-service-4.0)
+	- While registering the bot, use `https://<your_ngrok_url>/api/messages` as the messaging endpoint.
+
+    > NOTE: When you create your app registration, you will create an App ID and App password - make sure you keep these for later.
+
+3. Setup NGROK
+      - Run ngrok - point to port 3978
+
+	```bash
+	 #ngrok http -host-header=rewrite 3978
+	```   
+4. Setup for code
+
+  - Clone the repository
+    ```bash
+    git clone https://github.com/OfficeDev/Microsoft-Teams-Samples.git
+    ```
+   - Modify the `/appsettings.json` and fill in the following details:
```

Comment:
First mention the project path for which we need to update settings.

Reply:
fixed